modules/tinymce/src/core/main/ts/html/Sanitization.ts

```diff
@@ -239,21 +239,40 @@ const sanitizeMathmlElement = (node: Element, settings: DomParserSettings) => {
     return hasAllowedEncodings && Type.isString(encoding) && Arr.contains(allowedEncodings, encoding);
   };
 
-  purify.addHook('uponSanitizeElement', (node, evt) => {
-    const lcTagName = evt.tagName ?? node.nodeName.toLowerCase();
-
+  const isValidElementOpt = (lcTagName: string) => {
```

Comment:
This looks like this this could just return a boolean rather than an Optional wrapped boolean

Reply:
It's a bit of a tri-state, "Keep", "Remove", and "Don't change". I am concerned that the value on the object could be a true/false/undefined and that squashing it to true/false could make some changes to how the library act since it may go from a does-not-exist state to definitively-false state ( as might be the result when accepting the current value ). Using optionals removes this situation, which is why I picked it here.